### PR TITLE
hide UserInformation button

### PR DESCRIPTION
**Pull request by anistouri**

### CODE REVIEW

File: src/components/topBar/tests/TopBar.test.tsx [hunk 1]
```diff
@@ -24,12 +24,14 @@ const apps: Metadata[] = [
...
         appColor: 'blue',
         hiddenInAppsMenu: false,
+        hiddenUserInformation: false,
     },
     {
         name: 'App2',
         url: '/app2',
         appColor: 'green',
         hiddenInAppsMenu: true,
+        hiddenUserInformation: false,
```
Comment (AbdelHedhili) [inline src/components/topBar/tests/TopBar.test.tsx:34]: could set one with true to test both case no ?

File: src/components/topBar/AboutDialog.tsx [hunk 2]
```diff
@@ -275,14 +274,14 @@ function Module({ type, name, version, gitTag }: GridSuiteModule) {
...
-    globalVersionPromise,
     appName,
     appVersion,
     appGitTag,
     appLicense,
-    additionalModulesPromise,
     logo,
+    onClose,
+    globalVersionPromise,
+    additionalModulesPromise,
```
Comment (AbdelHedhili) [inline src/components/topBar/AboutDialog.tsx:284]: why move these lines, the file isn't changed.

**Review by AbdelHedhili — CHANGES_REQUESTED** [review]

File: src/components/topBar/AboutDialog.tsx (hunk 2 above)
Comment (anistouri) [reply src/components/topBar/AboutDialog.tsx]: done

File: src/components/topBar/tests/TopBar.test.tsx (hunk 1 above)
Comment (anistouri) [reply src/components/topBar/tests/TopBar.test.tsx]: Done

File: src/components/topBar/AboutDialog.tsx [hunk 3]
```diff
@@ -127,19 +127,18 @@ export type GridSuiteModule = {
     type: ModuleType;
     version?: string;
     gitTag?: string;
-    // license?: string;
```
[the remark this answers is not in the document]
Comment (anistouri) [reply src/components/topBar/AboutDialog.tsx:130]: It's a commented line that doesn't make sense

**Review by AbdelHedhili — APPROVED** [review]

**Pull request merged**